rds/iam.tf

```diff
@@ -1,8 +1,11 @@
+data "aws_caller_identity" "current" {}
```

Comment:
⚠️ could this lead to an issue where we apply changes to the _wrong_ account, because we haven't switched the AWS profile locally when we work through multiple projects/accounts?

Reply:
Commonly, we specify the profile directly within the provider block so it is rare to use the wrong profile there.

The issues of the wrong profile frequently happen in the deployment script (eg. `ecs-deploy` folder) which we usually skip the profile setting there. 

Overall, I think the wrong account issues are avoidable in these folders/modules by setting up the profile. Does it assure you? or does it happen to u in the other situation that I don't foresee?

Reply:
Good thanks! Then fine to clean up everywhere except for the deployment modules. WDYT?